docs/index.rst

```diff
@@ -0,0 +1,21 @@
+.. Python Zeppelin documentation master file, created by
+   sphinx-quickstart on Mon May 22 13:57:54 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Python Zeppelin's documentation!
+===========================================
+Test
```

Comment:
Not sure you meant to leave this here or you forgot; could you please add a description of the project here?